pkg/pool-weighted/contracts/lbp/LBPoolFactory.sol

```diff
@@ -27,30 +22,24 @@ import { LBPool } from "./LBPool.sol";
  * @dev This is a factory specific to LBPools, allowing only 2 tokens.
  */
 contract LBPoolFactory is IPoolVersion, ReentrancyGuardTransient, BasePoolFactory, Version {
-    using SafeERC20 for IERC20;
-    using SafeCast for uint256;
-
     // LBPs are constrained to two tokens.
     uint256 private constant _NUM_TOKENS = 2;
 
     string private _poolVersion;
 
     address internal immutable _trustedRouter;
-    IPermit2 internal immutable _permit2;
 
     constructor(
         IVault vault,
         uint32 pauseWindowDuration,
         string memory factoryVersion,
         string memory poolVersion,
-        address trustedRouter,
-        IPermit2 permit2
+        address trustedRouter
     ) BasePoolFactory(vault, pauseWindowDuration, type(LBPool).creationCode) Version(factoryVersion) {
         _poolVersion = poolVersion;
 
         // LBPools are deployed with a router known to reliably report the originating address on operations.
```

Comment:
Consider elaborating a bit: e.g., 
// This is used to ensure that only the owner can add liquidity to an LBP (including on initialization).